litebird_sim/detectors.py

```diff
@@ -132,6 +132,16 @@ class DetectorInfo:
              rotation at all, i.e., the detector is aligned with the
              boresight direction.
 
+        - pol_angle_rad (float): polarization angle of the detector in radiants. The default is 0.
```

Comment:
I would explain this better: “polarization angle of the detector with respect to the x-axis of the reference frame of the detector”.